web-src/src/admin/store/script-config-module.js

```diff
@@ -80,7 +80,21 @@ export default {
                     }
                     throw e;
                 });
-        }
+        },
+
+        deleteScript({}, {scriptName}) {
+            return axiosInstance.delete(`admin/scripts/${encodeURIComponent(scriptName)}`).then(() => {
```

Comment:
could you move `.then` on a new line please? I think starting every new operation on a new line makes code a bit easier to read

Reply:
yes sure. Are there maybe even some ESLinter options defined? Then I could simply let Prettier handle all the formatting.